### PR TITLE
Rpminfo

### PR DESCRIPTION
**Pull request by PetzJohannes**
Hi,

I am using the [rpmifno](https://www.php.net/manual/de/ref.rpminfo.php) library and PhpStorm does not provides autocompletion for this package.

So I decided to create my first stub contribution.

Here it is. Hopefully it is correct and I did not forget something. I can update my pull request if something is missing.

I executed the tests and this was my result on the master branch

```
Tests: 28163, Assertions: 97487, Failures: 13, Skipped: 51.
```
and this my results on the `rpminfo` branch
```
Tests: 28163, Assertions: 97487, Failures: 13, Skipped: 51.
```

I hope it is okay to open this pull request without green tests. :) 

Cheers Johannes

### CODE REVIEW

File: rpminfo/rpminfo.php [hunk 1]
```diff
@@ -0,0 +1,357 @@
...
+ * Path to the RPM file.
+ * </p>
+ * @param bool $full [optional] <p>
+ * If TRUE all information headers for the file are retrieved, else only a minimal set.
+ * </p>
+ * @param string $error [optional] <p>
+ * If provided, will receive the possible error message, and will avoid a runtime warning.
+ * </p>
+ *
+ * @return array|false <p>
```
[the remark this answers is not in the document]
Comment (PetzJohannes) [reply rpminfo/rpminfo.php]: changed the return information

File: rpminfo/rpminfo.php [hunk 2]
```diff
@@ -0,0 +1,357 @@
...
+ * Retrieve information about an installed package, from the system RPM database.
+ *
+ * @param string $nevr <p>
+ * Name with optional epoch, version and release.
+ * </p>
+ * @param bool $full [optional] <p>
+ * If TRUE all information headers for the file are retrieved, else only a minimal set.
+ * </p>
+ *
+ * @return array|false <p>
```
[the remark this answers is not in the document]
Comment (PetzJohannes) [reply rpminfo/rpminfo.php]: changed the return information

File: rpminfo/rpminfo.php [hunk 3]
```diff
@@ -0,0 +1,357 @@
...
+ * @param int $rpmmire [optional] <p>
+ * Pattern type, one of RPMMIRE_* constant.
+ * When < 0 the criterion must equals the value, and database index is used if possible.
+ * </p>
+ * @param bool $full [optional] <p>
+ * If TRUE all information headers for the file are retrieved, else only a minimal set.
+ * </p>
+ *
+ * @return array <p>
+ * An array of arrays of information or <b>FALSE</b> on error.
```
[the remark this answers is not in the document]
Comment (PetzJohannes) [reply rpminfo/rpminfo.php]: changed the return information

**Comment by PetzJohannes** [comment]
Hi @bzixilu thanks for the review.
I changed the suggested return values.
Can you please check it again?

**Comment by bzixilu** [comment]
Now everything seems good. Merged.
Thank you for your contribution!

**Pull request merged**